packages/fxa-auth-server/lib/senders/email.js

```diff
@@ -431,14 +431,13 @@ module.exports = function (log, config, bounces) {
     const translator = this.translator(message.acceptLanguage);
     message.layout = message.layout || 'fxa';
 
-    const { html, text, subject } = await this.fluentLocalizer.localizeEmail(
+    const { html, text } = await this.fluentLocalizer.localizeEmail(
```

Comment:
I think we still need `subject` out of this so we can pass it into that `emailConfig` object further down. I'm not sure what we use it for though, maybe for metrics somewhere? (I [shrugged in this comment](https://github.com/mozilla/fxa/pull/11962#discussion_r808290069) regarding it)